README.md

```diff
@@ -89,112 +89,27 @@ directory (such as Jupyter Lab notebooks) will persist.
-
-# Pull the Docker image from the public ECR Gallery
-docker pull public.ecr.aws/sagemaker/$ECR_GALLERY_IMAGE_ID
-
-# Tag the image for your private ECR Repository.
-# Define the following variables specific to your setup.
-export ECR_PRIVATE_REPOSITORY_NAME='...'
-export ECR_IMAGE_TAG='...'
-export AWS_ACCOUNT_ID='...'
-export AWS_ECR_REPOSITORY_REGION='...'
-export ECR_IMAGE_URI=$AWS_ACCOUNT_ID.dkr.ecr.$AWS_ECR_REPOSITORY_REGION.amazonaws.com/$ECR_PRIVATE_REPOSITORY_NAME:$ECR_IMAGE_TAG
-docker tag public.ecr.aws/sagemaker/$ECR_GALLERY_IMAGE_ID $ECR_IMAGE_URI
-
-# Push the image to your private repository
-docker push $ECR_IMAGE_URI
-
-# Step 1.1: Create an Amazon SageMaker Image.
-aws sagemaker create-image \
-    --image-name $SAGEMAKER_IMAGE_NAME \
-    --role-arn $SAGEMAKER_STUDIO_IAM_ROLE_ARN
-
-# Step 2: Create an Amazon SageMaker Image Version.
-aws sagemaker create-image-version \
-    --image-name $SAGEMAKER_IMAGE_NAME \
-    --base-image $ECR_IMAGE_URI
-
-# Optional, step 2.1: Verify that the Image and Image Version were successfully created.
-# You may have to specify a different `--version-number` if you created multiple Image Versions.
-aws sagemaker describe-image-version \
-    --image-name $SAGEMAKER_IMAGE_NAME \
-    --version-number 1
-
-# Step 3.1: Create a file that will contain App Image Config parameters.
-cat > /tmp/app-config.json << EOF
-{
-   "AppImageConfigName": "app-image-config-$SAGEMAKER_IMAGE_NAME",
-   "KernelGatewayImageConfig": { 
-      "FileSystemConfig": { 
-         "DefaultGid": 100,
-         "DefaultUid": 1000,
-         "MountPath": "/home/sagemaker-user"
-      },
-      "KernelSpecs": [ 
-         { 
-            "DisplayName": "Python 3 (ipykernel)",
-            "Name": "python3"
-         }
-      ]
-   }
-}
-EOF
-
-# Step 3.2: Create an Amazon SageMaker App Image Config.
-aws sagemaker create-app-image-config \
-    --cli-input-json file:///tmp/app-config.json
-
-# Step 4.1: Create a file that will contain default parameters for User Settings.
-cat > /tmp/default-user-settings.json << EOF
-{
-    "DefaultUserSettings": {
-        "KernelGatewayAppSettings": {
-            "CustomImages": [
-                {
-                    "ImageName": "$SAGEMAKER_IMAGE_NAME",
-                    "AppImageConfigName": "app-image-config-$SAGEMAKER_IMAGE_NAME",
-                    "ImageVersionNumber": 1
-                }
-            ]
-        }
-    }
-}
-EOF
-
-# Step 4.2: Update Amazon SageMaker Domain with the new default User Settings.
-aws sagemaker update-domain \
-    --domain-id $SAGEMAKER_STUDIO_DOMAIN_ID \
-    --cli-input-json file:///tmp/default-user-settings.json
-```
-
-Now, for your User Profile, log-out and log back in to start using the custom image.
+To use the sagemaker-distribution image in SageMaker Studio, simply go to the image selection dropdown, and select `SageMaker Distribution v{Major_version} {CPU/GPU}`.
```

Comment:
The image selection drop-down might change in future. It'd be better if we just give the link to a good AWS documentation page.